Makefile

```diff
@@ -263,6 +267,35 @@ image.%:
 	NRI_IMAGE_TAR=`realpath "$(IMAGE_PATH)/$${tag}-image-$${NRI_IMAGE_ID}.tar"`; \
 	$(DOCKER) image save "$${NRI_IMAGE_REPOTAG}" > "$${NRI_IMAGE_TAR}";
 
+.PHONY: licenses
+licenses: $(foreach p,$(PLUGINS),licenses.$(p)) $(foreach p,$(BINARIES),licenses.$(p))
+
+licenses.nri-resource-policy-% \
+licenses.nri-% \
+licenses.%:
+	$(Q)mkdir -p $(LICENSE_PATH); \
+	case $@ in \
+	    *.nri-resource-policy-*) \
+		dir=$(patsubst licenses.nri-resource-policy-%,./cmd/plugins/%,$@); \
+	        ;; \
+	    *.nri-*) \
+		dir=$(patsubst licenses.nri-%,./cmd/plugins/%,$@); \
+	        ;; \
+	    *.*) \
+		dir=$(patsubst licenses.%,./cmd/%,$@); \
+	        ;; \
+	esac; \
+	bin="$(patsubst licenses.%,%,$@)"; \
+	out="$(LICENSE_PATH)/$$bin"; \
+	rm -fr "$$out" && \
+	go-licenses save "$$dir" \
+	    --ignore github.com/containers/nri-plugins \
+	    --save_path "$$out" && \
+	go-licenses report "$$dir" \
+	        --ignore github.com/containers/nri-plugins \
+	        > "$$out"/licenses.csv && \
+	cat "$$out"/licenses.csv | sed 's/^.*,//g' | sort -u > "$$out"/licenses-summary.txt
```

Comment:
suggestion: could we put this longish script as a separate script under (`scripts/`, for example)? Would make it so much easier to read and maintain. WDYT?